orbeon/tests/test_orbeon_common.py

```diff
@@ -19,7 +19,6 @@
 #
 ##############################################################################
 from odoo.tests.common import TransactionCase
-from odoo import fields, models
```

Comment:
Some unused imports

Reply:
> Without going too much in depth of what these testcases validate. Is there a specific reason you use a Transaction case [here](https://github.com/open2bizz/orbeon/pull/2/files#diff-16482addd6e3a675992f014905898007L37) as base class? While sufficient, a SavepointCase would improve the overall performance of these tests since it only rolls back a transaction just until right after setUpClass has completed. (https://github.com/odoo/odoo/blob/12.0/odoo/tests/common.py#L442)

Thanks for you suggestions!
At that time I didn't noticed the `SavepointCase`. So good spotted!
Because I don't maintain this project anymore, I leave your suggestions for the current maintainer(s).